tests/test_job_management.py

```diff
@@ -1072,7 +1075,21 @@ def test_email_submitted_same_job(dispatcher_live_fixture, dispatcher_local_mail
     
     dispatcher_job_state.assert_email(state="submitted", number=2)
     dispatcher_local_mail_server.assert_email_number(2)
-    
+
+    # check the time in the title and filename is still the one of the first request
+    # check the time in the title and filename is still the one of the first request
+    for msg in dispatcher_local_mail_server.local_smtp_output:
+        msg_data = email.message_from_string(msg['data'])
+        assert msg_data[
+                   'Subject'] == f"[ODA][submitted] dummy first requested at {time_request_str} job_id: {dispatcher_job_state.job_id[:8]}"
```

Comment:
```suggestion
                   'Subject'] == f"[ODA][submitted] dummy requested at {time_request_str} job_id: {dispatcher_job_state.job_id[:8]}"
```
user expects it's first requested anyway, I think